tests/synpress.config.ts

```diff
@@ -0,0 +1,72 @@
+import { Wallet, utils } from 'ethers';
+import { defineConfig } from 'cypress';
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
+import synpressPlugins from '@synthetixio/synpress/plugins';
+
+const ethRpcUrl = process.env.NEXT_PUBLIC_LOCAL_ETHEREUM_RPC_URL;
```

Comment:
Might want to throw if `ethRpcUrl` or `arbRpcUrl` is undefined.

Reply:
I added some check at the top (same for private key)